Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -392,7 +412,10 @@
         return;
     }
 
-    m_webPageProxy.sendWithAsyncReply(Messages::DrawingArea::DispatchAfterEnsuringDrawing(), WTFMove(callbackFunction), m_identifier.toUInt64());
+    auto aggregator = CallbackAggregator::create(WTFMove(callbackFunction));
+    m_webPageProxy.sendWithAsyncReply(Messages::DrawingArea::DispatchAfterEnsuringDrawing(), [aggregator] { }, m_identifier);
+    for (auto& process : m_processesWithRegisteredRemoteLayerTreeDrawingAreaProxyMessageReceiver)
```

Comment:
We'll need to finesse how to drive rendering updates for subprocesses. We want to avoid doing anything to a process for a frame which has nothing to do.

Reply:
In this case I'm pretty sure we need to ensure all processes have drawn, but there will be some places where we will want to only do things when there are things to do.